main.py

```diff
@@ -41,15 +44,68 @@ def main(args_cli):
 
     # unifies whatever a user specified as paths into a list of paths
     video_paths = form_list_from_user_input(args.video_paths, args.file_with_video_paths, to_shuffle=True)
-
     print(f'The number of specified videos: {len(video_paths)}')
 
-    for video_path in tqdm(video_paths):
-        extractor._extract(video_path)  # note the `_` in the method name
+    return extractor, video_paths, args
 
-    # yep, it is this simple!
 
+def main(extractor, args, video_paths):
+    for video_path in tqdm(video_paths):
+        extractor._extract(video_path)  # note the `_` in the method name
 
 if __name__ == '__main__':
     args_cli = OmegaConf.from_cli()
-    main(args_cli)
+    _extractor, _vp, args = get_extractor_and_params(args_cli)
```

Comment:
tries to initialize the model on a cuda device here. since the submitit is usually run on a login node, it will fail with an error that cuda device is not available.

how do you run it on your end for it to pass this error?